src/services/etherpad/hooks/Pad.js

```diff
@@ -58,7 +58,7 @@ const before = {
 	find: [disallow()],
 	get: [disallow()],
 	create: [
-		globalHooks.hasPermission('TOOL_CREATE'),
+		globalHooks.hasPermission('TOOL_CREATE_ETHERPAD'),
```

Comment:
how / where students from creating Etherpads in course, i.e. allow only in course group?

Reply:
The student enters the course and creates groups within the course. And within this group, he creates a topic and adds an etherpad to it. 
That's all he can do. Create a topic inside the group. 
Instead, the teacher can create a topic and add an etherpad to it inside the course and inside the group. 

With the previous TOOL_CREATE permission, the teacher was fine, but the student has more rights when he is given this permission. He can add conferences on the TOOLS tab. And we don't need this.

Translated with www.DeepL.com/Translator (free version)